src/config/dynamic-theme-fixes.config

```diff
@@ -13357,6 +13357,60 @@ INVERT
 
 ================================
 
+landscan.ornl.gov
```

Comment:
As for maps.cz it seems it works nice but for this map is not.

Have you zoomed map to some city when default landscan is enabled?
Without your fix it's like this:
![obraz](https://github.com/darkreader/darkreader/assets/56877029/31753d71-f684-4f33-8b59-470ca6aadb8e)

but after updates it is more eye-painfull:
![obraz](https://github.com/darkreader/darkreader/assets/56877029/2a1676d8-507c-4dad-8654-b281fdd0a1a6)

Not sure if I would like to touch this map at all.